src/basic/conf-files.c

```diff
@@ -123,6 +123,27 @@ static int base_cmp(char * const *a, char * const *b) {
         return strcmp(basename(*a), basename(*b));
 }
 
+static int get_sorted_files_from_hashmap(Hashmap *fh, char ***ret) {
```

Comment:
Maybe name this `copy_sorted_files_from_hashmap()` to more clearly indicate that we copy the files strv?